tjreports/languages/administrator/en-GB/en-GB.com_tjreports.ini

```diff
@@ -111,6 +111,8 @@ COM_TJREPORTS_ACCESS_VIEW_SUBSCRIBE_DESC="Allow user to view only subscribed rep
 JACTION_VIEW="View"
 JACTION_VIEW_COMPONENT_DESC="Allow user to view reports in this extension"
 JACTION_EXPORT="Export"
+COM_TJREPORTS_ACCESS_PIIDATA="PII Data"
```

Comment:
change values as 
`View PII Data`
&
`Allow user to see Personally Identifiable Information (PII) data on the report.`